### PR TITLE
tests(*): add more debug logs (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by chronolaw**

### CODE REVIEW

File: lualib/resty/events/worker.lua [hunk 1]
```diff
@@ -331,6 +335,8 @@ function _M:publish(target, source, event, data)
     assert(type(source) == "string" and source ~= "", "source is required")
     assert(type(event) == "string" and event ~= "", "event is required")
 
+    log(DEBUG, "source=", source, ", event=", event, ", data=", cjson_encode(data))
```
Comment (dndx) [inline lualib/resty/events/worker.lua:338]: Please add some prefix to the log message.

File: lualib/resty/events/callback.lua [hunk 2]
```diff
@@ -110,9 +110,18 @@ function _M:do_event(d)
     local event  = d.event
     local data   = d.data
     local wid    = d.wid
+    local time   = d.time
+
+    ngx.update_time()
+    local now = ngx.now()
+
+    if time then
+      log(DEBUG, "worker-events: receive event; source=", source,
```
Comment (dndx) [inline lualib/resty/events/callback.lua:119]: Could you add the payload to other logs as well?

Comment (chronolaw) [reply lualib/resty/events/callback.lua]: done.

File: lualib/resty/events/worker.lua (hunk 1 above)
Comment (chronolaw) [reply lualib/resty/events/worker.lua]: done.

**Comment by chronolaw** [comment]
Closed for no activity.

**Pull request closed without merging**